scss/1-settings/_colour-language.scss

```diff
@@ -29,9 +29,13 @@ $cads-language__warning-colour: palette.$cads-palette__red !default;
 $cads-language__success-colour: palette.$cads-palette__green !default;
 
 // Navigation colours
+$cads-language__nav-colour: palette.$cads-palette__white !default;
 $cads-language__nav-background-colour: $cads-language__brand-primary !default;
 $cads-language__nav-hover-background-colour: palette.$cads-palette__heritage-blue-dark !default;
 $cads-language__nav-hover-border-colour: palette.$cads-palette__heritage-blue-dark !default;
+$cads-language__nav-focus-colour: $cads-language__link-colour !default;
```

Comment:
I'm slightly nervous about the focus-colour being the link-colour. I'm not sure that we will remember to update the colour for advisernet and intranet. I don't think the vrts in the public-website side will catch it.
The rest of the new colours look good to me because they are quire consistent throughout products.

Reply:
oh yeah, maybe this should use the palette colour instead

Reply:
Moved this to use the palette colour, but double checked and this doesn't have much affect on intranet and advisernet as we don't override these values. Makes sense to use the palette colour directly still though 👍

Reply:
Won't using a palette colour still display the nav colour on focus as blue in intranet and advisernet?

Reply:
Yep, but that's what we want. We don't override the colour for those products.

Reply:
For our future selves: We got to the bottom of this. The focus colour was previously using the _brand_ language colour which _does_ get overridden per-product typically. Have updated it to use that colour for backwards compatibility.